# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sudo/sudo-group.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sudo-1.9.13p2/plugins/sudoers/sudoers.in.vanilla	2023-02-28 16:32:49.746594423 +0100
+++ sudo-1.9.13p2/plugins/sudoers/sudoers.in	2023-02-28 16:32:59.357594952 +0100
@@ -88,7 +88,7 @@
 # %wheel ALL=(ALL:ALL) NOPASSWD: ALL
 
 ## Uncomment to allow members of group sudo to execute any command
-# %sudo	ALL=(ALL:ALL) ALL
+%sudo	ALL=(ALL:ALL) ALL
 
 ## Uncomment to allow any user to run sudo if they know the password
 ## of the user they are running the command as (root by default).
